app/cdap/components/WrangleHome/Components/WrangleCard/Components/WidgetData/index.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import React from 'react';
+import { findHighestVersion } from 'services/VersionRange/VersionUtilities';
+import reject from 'lodash/reject';
+import ImageOutlined from '@material-ui/icons/ImageOutlined';
+import { Avatar } from '@material-ui/core';
+
```

Comment:
exported functions need jsdoc comments - especially complicated ones

Reply:
Also, it looks like this is a copy of https://github.com/cdapio/cdap-ui/blob/develop/app/cdap/components/Connections/Create/reducer.ts#L170 -- this is fine for now, but please include a reference in the JSDoc (e.g. with `@see`) to the original location.